vocabularies/UI.xml

```diff
@@ -1285,13 +1285,17 @@ The trend is
         </Member>
       </EnumType>
 
-      <!-- Under discussion
-        <Term Name="DisplayTimeZone" Type="?Edm.Int32?" AppliesTo="Property">
-        <Annotation Term="Core.Description">
-        <String>Contains information for time- or date-time-fields in which time zone the time value should be displayed.</String>
+      <Term Name="DisplayTimezoneOffset" Type="Edm.String" AppliesTo="Property">

```

Comment:
```suggestion
      <Term Name="DisplayTimezoneOffset" Type="Edm.String" Nullable="false" AppliesTo="Property">

```
Or is this really nullable?

Reply:
What if the timestamp field is null? Do we still require a non-null offset?